src/dotnet/commands/dotnet-add/dotnet-add-package/xlf/LocalizableStrings.pl.xlf

```diff
@@ -14,7 +14,7 @@
         <note />
       </trans-unit>
       <trans-unit id="AppHelpText">
-        <source>Package references to add</source>
+        <source>Package reference to add</source>
         <target state="translated">Odwołania do pakietów, które mają zostać dodane</target>
```

Comment:
(not related to this PR) this translation couldn't be any worse cc: @blackdwarf - I just looked at the Polish translation and it is quite not right - while literally it is translated fairly ok, semantically and contextually it doesn't make sense